docusaurus-plugin-moonwave/src/index.js

```diff
@@ -3,6 +3,86 @@ const fs = require("fs")
 const { promisify } = require("util")
 const exec = promisify(require("child_process").exec)
 
+const mapLinks = (nameSet, items) =>
+  items.map((name) => {
+    if (!nameSet.has(name)) {
+      throw new Error(
+        `Moonwave plugin: "${name}" listed in classOrder option does not exist`
+      )
+    }
+
+    return {
+      type: "link",
+      href: `/api/${name}`,
+      label: name,
+    }
+  })
+
+function parseSimpleClassOrder(content, classOrder, nameSet) {
+  const listedLinks = mapLinks(nameSet, classOrder)
+
+  const unlistedLinks = content
```

Comment:
These names were really long! Now they're nice and short. Since we extracted this out into a function, we can use more general variable names because we can get context from the function name.